skstats/hypotests/fit_utils/api_check.py

```diff
@@ -0,0 +1,73 @@
+# API checks
```

Comment:
Can you add a module-level docstring explaining all checks in here? Seems a neat thing to do. It should also tell the reader whether this is user-face code or more internal business ...